### PR TITLE
updated to internals to allow websocket passthrough

### PR DESCRIPTION
**Pull request by kwhitley**

### CODE REVIEW

File: examples/websocket/Room.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    // add socket.server to list of open sockets
+    sockets.push({ id, server })
+
+    // send a message to all connected sockets
+    this.sendMessage('A new connection has been established!')
+
+    return new Response(null, { status: 101, webSocket: client })
+  }
+
+  sendMessage(message: string) {
```
Comment (gishmel) [inline examples/websocket/Room.ts]: Think broadcast is a better name for this function.

**Comment by gishmel** [comment]
~~I am still receiving the error as if the Upgrade header is not being passed through but I guess it's also possible I am not doing something else properly?~~
Screenshot from 2023-01-11 13-42-12

Turns out case sensitivity will get me everytime; Connection should be Upgrade and it works swimmingly! Thanks so much for the quick fix!

**Comment by gishmel** [comment]
Turns out case sensitivity will get me everytime; Connection should be Upgrade and it works swimmingly! Thanks so much for the quick fix!

File: examples/websocket/Room.ts (hunk 1 above)
Comment (kwhitley) [reply examples/websocket/Room.ts]: Don't disagree (thankfully it's just an example)!  I'll make the changes :p

**Comment by kwhitley** [comment]
Wait, is there a case-sensitivity issue going on?  Either should be fine, surely?

**Comment by gishmel** [comment]
I thought it was a case sensitivity issue but now I am getting it to work with either so idk must be me haha. Anyways it totally works getting back a good ole 101 status code and have a websocket to work with! Thanks for all your help!

**Comment by kwhitley** [comment]
Whew! That's great news! :)  Happy to help - and your issue/description/investigation definitely helped me get to the finish line faster!

**Pull request merged**